core/src/test/java/nu/yona/server/analysis/service/ActivityServiceTests.java

```diff
@@ -118,19 +131,135 @@ public void setUp()
 	{
 		return Collections.singletonMap(Locale.forLanguageTag("en-US"), string);
 	}
+	
+	@Test
+	public void dayActivityOverview()
+	{
+		ZonedDateTime today = getDayStartTime(ZonedDateTime.now(userAnonZone));
+		ZonedDateTime yesterday = today.minusDays(1);
+		
+		// gambling goal was created 2 weeks ago, see above
+		// mock some activity in previous week on yesterday 20:58-21:00
```

Comment:
You say "in previous week", but yesterday isn't necessarily in a previous week, right?